Tutorials update (#171)
immunomind/immunarch: vignettes/v2_data.Rmd

For parsing IgBLAST results process the data with MigMap first.

You can load the data either from a single file, a list of repertoire file paths or from a folder with repertoire files. 

**Working with your own files**

If you have your own files, you should just specify a path to your file or to a folder with files. Then load data using `repLoad`:

```{r, eval=F}
#path argument is a path to the folder with your file or files including metadata file also(if you have one).
immdata <- repLoad(path)
```

**Working with example files**

You could find a folder with example files [here](https://github.com/immunomind/immunarch/tree/master/inst/extdata/io) and use it to test data loading without your own files. 

If you are not familiar with file paths, you can download example data to your working directory.
You can obtain you working directory with `getwd()` command
```{r, eval=F}

> getwd()

Reviewer: Looks like this lacks the closing for the code snippet

For parsing IgBLAST results process the data with MigMap first.

> You can load the data either from a single file, a list of repertoire file paths or from a folder with repertoire files. 

Reviewer 2: _... data from a single file, a list of repertoire file paths, or from a folder with repertoire files.

> **Working with own files**

Reviewer 3: ```suggestion
**Working with your own files**
```

> **Working with own files**

Reviewer 2: whose? "Working with your files" or "Working with user's files"

> If you have your own files, you should just specify a path to your file or to a folder with files. Then load data using `repLoad`:

Reviewer 2: ...files, just specify the path to your file...

```{r, eval=F}

> #a path to own folder or file

Reviewer 3: ```suggestion
#path argument is a path to the folder with your file or files including metadata file also(if you have one).
```

> #a path to own folder or file

Reviewer 2: same as before: your/user's

immdata <- repLoad(path)
```

> **Working with example files**

Reviewer 2: sample files (instead of example). here and in the nest few sentences.

> You could find the path to a folder with example files [here](https://github.com/immunomind/immunarch/tree/master/inst/extdata/io). 

Reviewer 3: ```suggestion
You could find a folder with example files [here](https://github.com/immunomind/immunarch/tree/master/inst/extdata/io) and use it to test data loading without your own files. 
```

> 

Reviewer 2: +++

> Let's check your working folder

Reviewer 2: working directory (here and in the following sentences)

> Let's check your working folder
> ```{r, eval=F}
> getwd()
> ```
> 
> Let's download file `'Sample1.tsv.gz'` in the working folder from [here](https://github.com/immunomind/immunarch/tree/master/inst/extdata/io) and load the file:
> 
> ```{r, eval=F}
> immdata <- repLoad('Sample1.tsv.gz')
> ```

Reviewer 3: ```suggestion
If you are not familiar with file paths, you can download example data to your working directory.
You can obtain you working directory with `getwd()` command
```{r, eval=F}
getwd()
```
If you have example files in your working directory you can use the next chunk to load example data with repLoad

```{r, eval=F}
immdata <- repLoad('Sample1.tsv.gz')
```

> You could also download all files to the `'example'` folder and load all of them:

Reviewer 3: ```suggestion
You could also download all files to the `'example'` folder in your working directory and load all of them by passing folder name to repLoad function in a quotes:
```

```{r, eval=F}
immdata <- repLoad('example')
```

> Also, example data is downloaded with `immunarch` packages. You could load whole example data using the following command:

Reviewer 3: ```suggestion
Also, example data are already downloaded with `immunarch` package. You could load whole example data using the following command:
```

> Also, example data is downloaded with `immunarch` packages. You could load whole example data using the following command:

Reviewer 2: You can download all sample files

```{r, eval=F}

> #path to folder with example data

Reviewer 2: ...to the folder...
